Kernel/Process.cpp

```diff
@@ -452,20 +452,36 @@ void create_signal_trampoline()
     g_signal_trampoline_region->remap();
 }
 
-void Process::crash(int signal, FlatPtr ip, bool out_of_memory)
+void Process::crash(int signal, Optional<RegisterState const&> regs, bool out_of_memory)
 {
     VERIFY(!is_dead());
     VERIFY(&Process::current() == this);
 
+    auto ip = regs.has_value() ? (*regs).ip() : 0;
+
     if (out_of_memory) {
         dbgln("\033[31;1mOut of memory\033[m, killing: {}", *this);
     } else {
         if (ip >= kernel_load_base && g_kernel_symbols_available) {
             auto const* symbol = symbolicate_kernel_address(ip);
-            dbgln("\033[31;1m{:p}  {} +{}\033[0m\n", ip, (symbol ? symbol->name : "(k?)"), (symbol ? ip - symbol->address : 0));
+            dbgln("\033[31;1m{:p}  {} +{}\033[0m", ip, (symbol ? symbol->name : "(k?)"), (symbol ? ip - symbol->address : 0));
```

Comment:
Was the `\n` here superfluous independently of the change in this commit (since it's a `dbgln` call)? If so, I'd probably remove them in their own commit.

Reply:
Yeah, it made the formatting a bit nicer, but it's also fine with the newline, so I'll just keep the newline.